src/CustomPasswordExpiredController.js

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
Not sure if it matters. This is a new file, should be `Copyright (c) 2017` The first date you publish.